### ISSUE TITLE
Print / PDF export CSS scrollbars

### ISSUE THREAD

**Issue by rgbkrk** [nteract/archived-desktop-app#1893]
Cells should be expanded by default when printing (or so I assume). Otherwise it shows the scrollbars. That or maybe instead of scrollbars we show something to indicate there would be more content if it was expanded.

![screen shot 2017-09-14 at 10 57 55 am](https://user-images.githubusercontent.com/836375/30445660-9536a4a4-993b-11e7-8a3b-829f3ab0221e.png)

/cc @HenrikEckermann

**Comment by HenrikEckermann** [comment]
Hi Kelly,

Thank you very much!

Ok, so you think I can just print it?

For the images, I then will just wait until this problem is resolved because I very much like that editor.

By the way: Can you also avoid that there is this scrolling box just like Jupyter when I want to look at the whole table?

I hope you keep going.

Best regards,

Henrik

**Comment by rgbkrk** [comment]
We won't have these fixed in time for your current homework submissions. We'll keep at it to make this better though. 😄

**Comment by HenrikEckermann** [comment]
Yes I know. I meant I hope you generally keep going developing nteract because I like the idea and outlook.

I will keep an eye on it. I also try to use it. If I cannot do something with it, I guess I can just save the notebook and try in Jupyter instead.

**Comment by jkornblum** [comment]
Is this the same as #1610 or a different feature?

**Comment by rgbkrk** [comment]
D'oh! Of course. I think it's the same?

**Comment by captainsafia** [comment]
This might be tricky to do. Our current PDF-export strategy uses the browser "Print to PDF" strategy in Chrome. So we are limited here, especially in the scenario where the scroll is horizontal, not vertical.
